lib/link.ts

```diff
@@ -19,16 +22,35 @@ if (process.env.NODE_ENV === "development" || process.env.NODE_ENV === "test") {
 
 export const handleLink = async (url: string) => {
   if (!url) {
+    console.error("no url to handle");
     return;
   }
+  console.info("handling link", url);
 
   const parsedUrl = new URL(url);
   if (!parsedUrl.protocol) {
+    console.error("no protocol in URL", url);
     return;
   }
 
   if (SUPPORTED_SCHEMES.indexOf(parsedUrl.protocol) > -1) {
     let { username, hostname, protocol, pathname, search } = parsedUrl;
+    if (parsedUrl.protocol.startsWith("nostr+walletauth")) {
+      const nwaClient = nwa.NWAClient.parseWalletAuthUrl(url);
+
```

Comment:
Do `router.dismissAll()` + error handling

Reply:
I tried to add this but my Alby Go crashes. Not sure why?

Reply:
I added try-catch and also a go back since that doesn't work from the send page, but this feels super hacky